src/main.js

```diff
@@ -178,31 +178,44 @@ function codeReviewFeatures(config) {
         }
     }
 
-    const summarySelectors =
-        '#compare-diff-content, #pr-tab-content, #commit, #diff'
-    const diffSelector = 'section.bb-udiff'
+    const summarySelectors = [
+        '#compare-diff-content',
+        '#pr-tab-content',
+        '#commit',
+        '#diff',
+    ]
+    const diffSelector = '#diff'
+    const allSelectors = [...new Set([...summarySelectors, diffSelector])].join(
+        ', '
+    )
 
     // Have to observe the DOM because some sections
     // load asynchronously by user interactions
     // eslint-disable-next-line no-new
-    new SelectorObserver(
-        document.body,
-        [summarySelectors, diffSelector].join(', '),
-        function() {
-            try {
-                if (this.matches(summarySelectors)) {
-                    return manipulateSummary(this)
-                }
+    new SelectorObserver(document.body, allSelectors, function() {
+        if (
+            this.style.display === 'none' ||
+            this.getAttribute('aria-hidden') === 'true'
+        )
+            return
 
-                if (this.matches(diffSelector)) {
-                    return manipulateDiff(this)
-                }
-            } catch (error) {
-                // Something went wrong
-                console.error('refined-bitbucket(code-review): ', error)
+        try {
+            if (this.matches(summarySelectors.join(', '))) {
+                manipulateSummary(this)
             }
+
+            if (this.matches(diffSelector)) {
```

Comment:
looking for parent `diff` just to make sure it gets validated for `display:none` before running functions.